client/src/pages/ProfileSetupPage/ProfileSteps/ProfileStep3/AvailableTimesDropDown/AvailableTimesDropDown.js

```diff
@@ -0,0 +1,79 @@
+import React from 'react';
+import { Box, MenuItem, Select, Typography } from '@material-ui/core';
+
+const AvailableTimesDropDown = ({timeAvlFrom, timeAvlUntil, handleUserInput}) => (
+    <Box className="step3__content--hours">
+        <Typography variant="h6" className="step3__content--hours--title">Available Hours:</Typography>
+        <div className="step3__content--hours--values">
+            <Box className="step3__content--hours--select">
+            <Select
+            onChange={handleUserInput}
+            disableUnderline
+            name="timeAvlFrom"
+            value={timeAvlFrom}
+            >
+                <MenuItem value={0}>00:00</MenuItem>
```

Comment:
Could try to map this component over an array [0...24] to save some lines
something like
`{[... Array(24).keys()].map(t =>
                <MenuItem value={t}> {("00" + t).slice (-2) + ":00"}</MenuItem>)
              }`

Reply:
will do